.gitignore

```diff
@@ -36,7 +36,6 @@ cscope.out
 *.lck
 OldVersions/
 
-gui/build
```

Comment:
Why don't we need this anymore?

Reply:
It is already covered by the more general rule on line 8. And this is obsolete, as it would be `software/gui/build` now